# Analysing-single-cell-RNA-and-ATAC-sequence-data
▪ Integrated real-world single cell RNA and ATAC sequence data using unsupervised Manifold-learning based maximum mean discrepancy (MMD) measure to jointly embed RNA and ATAC sequences. 
▪ Evaluated data integration using FOSCTTM score – and applied hyper-parameter tuning to yield an average FOSCTTM score close to zero, for better alignment performance. 
▪ Performed dimensionality reduction by comparing Non-Matrix Factorization and Latent Dirichlet Allocation methods on genome sequences. 
▪ Clustered the data after dimensionality reduction using K-Means and used UMAP to visually check the clustering results, and finally evaluated if the separate RNA and ATAC are co-assayed and compared cluster overlap fraction using DICE Coefficient values. 
▪ Generated a graph to represent physical protein-protein interactions and performed Graph partitioning by comparing min-cut and normalized-cut algorithms. 
▪ Future work would include evaluating the partition performance by using Gene Ontology enrichment analysis to check number of enriched clusters out of 50 output clusters.
